### ISSUE TITLE
gmax funky still?

### ISSUE THREAD

**Issue by dbuona** [lizzieinvancouver/temporalvar#53]
@donahuem @lizzieinvancouver 
Hi guys, I think my confusion about gmax might also be present in the model, either that or I still don't understand it properly.
I'll explain....
If you run lines 80-81 in PriEff_Species.R you get vector of values for gmax in every year of 0.8 for both species which is good. Last time we met, you pointed out that a species could germinate less than this in any given year due to lack of chilling, which in my understanding, the parameter to describe this is the cumulative germination for each species at the end of the season, which I can access  with the function sapply(g_cumulative, tail, 1).

However, even when I basically set chilling to 0, I am still getting 80% germination in each of the runs. 

To further complicate, when I run lines 87-107 in  PriEff_Species.R, which are supposed to let maximum germination vary by species, the resulting vectors of gmax are different for each year of the a run, suggesting its not behaving as a species level parameter. 

The description above is probably is probably hard to follow--- but I think it suggests at our next meeting, it would probably be helpful to step through the model again, either to fix something that is off with the model or my understanding of it.

**Comment by lizzieinvancouver** [comment]
@dbuona If you can point me to @donahuem code that made all the plots for each run (or something like that, basically what she was using to visualize the model runs) then I can try to double-check we're looking at the right output.

**Comment by dbuona** [comment]
@lizzieinvancouver Thanks!  I just pushed a new commit with all of Megan's original plots turned back on (lines 55, 114, 159, 173 respectively) in PriEff_Species.R. And FYI, I have also set the "low chilling" scenarios extremely low in PriEff_Envt to try to generate more germination failures, to no avail.

**Comment by lizzieinvancouver** [comment]
@dbuona Okay, I just took a quick look ... and I agree that gmax seems like it should be the parameter we want (I think). Can you post Megan's plots from a super low and super high chill run here so I can look at them? I suspect we'll want them next week also. Thanks!

**Comment by dbuona** [comment]
yup, I added 4plots.
https://github.com/lizzieinvancouver/temporalvar/blob/priorityeff/plots/both_sps_fixed_fract_highchill.pdf
https://github.com/lizzieinvancouver/temporalvar/blob/priorityeff/plots/bothsps_same_fract_lowchill.pdf
https://github.com/lizzieinvancouver/temporalvar/blob/priorityeff/plots/sps_vary_frac_highchill.pdf
https://github.com/lizzieinvancouver/temporalvar/blob/priorityeff/plots/sps_vary_frac_lowchill.pdf

**Comment by lizzieinvancouver** [comment]
@dbuona Wondering if this is all sorted and we can close this issue?
